lib/fluent/system_config.rb

```diff
@@ -46,6 +46,13 @@ class SystemConfig
     config_param :rpc_endpoint, :string, default: nil
     config_param :enable_get_dump, :bool, default: nil
     config_param :process_name, default: nil
+    config_param :listen_port, :integer, default: nil
```

Comment:
Could you remove this line?
There's no need to configure system-wide `listen_port` because this configuration cannot be shared by anything.
Removing this line also makes `port_override` methods useless.